sys/include/psa_crypto/psa/crypto_sizes.h

```diff
@@ -562,6 +562,7 @@ extern "C" {
  *          parameters are incompatible.
  */
 #define PSA_CIPHER_FINISH_OUTPUT_SIZE(key_type, alg) \
+        ((key_type == PSA_KEY_TYPE_CHACHA20) ? 0 : 0 )
```

Comment:
Same here?

Reply:
problem here is that we do not know, how much data is left in the buffer. So we can only say "64 Bytes is sufficient"

Reply:
...which is still better than saying `0` :)